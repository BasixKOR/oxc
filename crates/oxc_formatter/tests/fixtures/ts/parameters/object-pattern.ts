(
  options,
  { log, logger, messenger }: {
    log: LogFun;
    logger: Logger;
    messenger: Messenger;
  }) => {

}