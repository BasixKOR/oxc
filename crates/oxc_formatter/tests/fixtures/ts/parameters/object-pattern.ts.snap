---
source: crates/oxc_formatter/tests/fixtures/mod.rs
---
==================== Input ====================
(
  options,
  { log, logger, messenger }: {
    log: LogFun;
    logger: Logger;
    messenger: Messenger;
  }) => {

}
==================== Output ====================
(
  options,
  {
    log,
    logger,
    messenger,
  }: {
    log: LogFun;
    logger: Logger;
    messenger: Messenger;
  },
) => {};

===================== End =====================
